daemon/src/db/closed.rs

```diff
@@ -142,6 +142,8 @@ impl Connection {
             if let Err(e) = fut.await {
                 tracing::warn!(order_id = %id, "Failed to move closed CFD: {e:#}");
             }
+
+            tracing::debug!(order_id = %id, "Moved CFD to `closed_cfds` table");
```

Comment:
This will log-unconditionally, so also in the case of an error above :)

Probably best to use a `match` or early return.